test/AutoDiff/protocol_requirement_autodiff.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-run-simple-swift
+// RUN: %target-swift-frontend -emit-sil -verify %s
```

Comment:
Not too familiar with the swift testing flags, but do we need the `-emit-sil` flag as these look like type checking tests.

Reply:
Yes I also wasn't sure. I looked into other tests and replaced it with `-typecheck` which seems to work. :)